src/sage/categories/finite_dimensional_algebras_with_basis.py

```diff
@@ -938,15 +940,15 @@ def is_identity_decomposition_into_orthogonal_idempotents(self, l):
 
             Here `1+x` and `-x` are neither idempotent nor orthogonal::
 
-                sage: A.is_identity_decomposition_into_orthogonal_idempotents([1+x,-x])
+                sage: A.is_identity_decomposition_into_orthogonal_idempotents([1 + x, -x])
                 False
 
             With the algebra of the `0`-Hecke monoid::
 
-                sage: from sage.monoids.hecke_monoid import HeckeMonoid
-                sage: A = HeckeMonoid(SymmetricGroup(4)).algebra(QQ)
-                sage: idempotents = A.orthogonal_idempotents_central_mod_radical()
-                sage: A.is_identity_decomposition_into_orthogonal_idempotents(idempotents)
+                sage: from sage.monoids.hecke_monoid import HeckeMonoid                 # optional - sage.groups
+                sage: A = HeckeMonoid(SymmetricGroup(4)).algebra(QQ)                    # optional - sage.groups sage.modules
+                sage: idempotents = A.orthogonal_idempotents_central_mod_radical()      # optional - sage.groups sage.modules sage.rings.number_field
+                sage: A.is_identity_decomposition_into_orthogonal_idempotents(idempotents)          # optional - sage.groups sage.modules sage.rings.number_field
                 True
```

Comment:
Why not align here?

Reply:
No problem.

Reply:
This super long identifier somehow stopped me from finding a better looking solution